docker-compose.yml

```diff
@@ -40,27 +41,27 @@ services:
       - POSTGRE_HOST=db
       - LABEL_STUDIO_HOST=${LABEL_STUDIO_HOST:-}
       - JSON_LOG=1
-#      - LOG_LEVEL=DEBUG
+    #      - LOG_LEVEL=DEBUG
     volumes:
       - ./mydata:/label-studio/data:rw
     command: label-studio-uwsgi
 
   db:
     image: postgres:11.5
     hostname: db
-    restart: always
+    restart: unless-stopped
     # Optional: Enable TLS on PostgreSQL
     # Just drop your server.crt and server.key into folder 'deploy/pgsql/certs'
     # NOTE: Both files must have permissions u=rw (0600) or less
-#    command: >
-#      -c ssl=on
-#      -c ssl_cert_file=/var/lib/postgresql/certs/server.crt
-#      -c ssl_key_file=/var/lib/postgresql/certs/server.key
+    #    command: >
+    #      -c ssl=on
+    #      -c ssl_cert_file=/var/lib/postgresql/certs/server.crt
+    #      -c ssl_key_file=/var/lib/postgresql/certs/server.key
     environment:
       - POSTGRES_HOST_AUTH_METHOD=trust
     volumes:
       - ${POSTGRES_DATA_DIR:-./postgres-data}:/var/lib/postgresql/data
       - ./deploy/pgsql/certs:/var/lib/postgresql/certs:ro
 
 volumes:
-  static: {}
```

Comment:
Actually this volume was deprecated, you can remove volumes section here and `static` from your override file